Initial draft of release notes for 2022-10 update

francine-vendia opened the pull request.

francine-vendia commented:
![GraphQL_Explorer_AutoFill_ID](https://user-images.githubusercontent.com/105127677/198167182-cfd9108c-492f-42c2-8791-2a0e13353bf0.png)

james-vendia commented on releases/2022-10-28-releasenotes.md:
```diff
@@ -0,0 +1,38 @@
+
+They're at it again! The Vendia Engineering team has a series of new improvements to continue advancing the experience building on Vendia.  
+
+As always, see our [Docs](https://www.vendia.com/docs/share) for the complete details of each new feature and our [Examples](https://github.com/vendia/examples) for more guidance. 
```
```suggestion
As always, see [our docs](https://www.vendia.com/docs/share) for the complete details of each new feature and [our examples](https://github.com/vendia/examples) for more guidance. 
```

james-vendia replied:
Take or leave...

francine-vendia replied:
I updated it in local branch and pushed - should be incorporated now

james-vendia commented on releases/2022-10-28-releasenotes.md:
```diff
@@ -0,0 +1,38 @@
+Questions? Please [contact us](https://www.vendia.com/contact-us).
+
+# We're COMical 
+While we are quite amusing, we are also officially on [Vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(Vendia.net)](https://vendia.net/) but now our website is live on Vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
```
```suggestion
While we are quite amusing, we are also officially on [vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(vendia.net)](https://vendia.net/) but now our website is live on vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
```

francine-vendia replied:
I updated it in local branch and pushed - should be incorporated now

james-vendia commented on releases/2022-10-28-releasenotes.md:
```diff
@@ -0,0 +1,38 @@
+# We're COMical 
+While we are quite amusing, we are also officially on [Vendia.com](https://www.vendia.com)! Previously we were nothing-but-net [(Vendia.net)](https://vendia.net/) but now our website is live on Vendia.com. Check it out yourself! Don't worry; development is still on [share.vendia.net](https://share.vendia.net).
+
+# Advancing Azure Beta Feature
```
I'd scrap this section.  Azure is still in beta and it's not clear from the description what's new/different/better.  My general rule is that if we can't show a concrete example, we shouldn't share it in external release notes.

francine-vendia replied:
Makes sense - removed.

james-vendia commented on releases/2022-10-28-releasenotes.md:
```diff
@@ -0,0 +1,38 @@
+# Advancing Azure Beta Feature
+Sharing, without limitation, is our ethos and we continue expanding the surface area on which to share. We've added validations and stabilization for sharing between two azure-backed nodes by resolving errors associated with metadata availability, handling mutations, and cross-cloud file sharing.
+
+# Ensuring Transaction Success
```
Can we / are we comfortable quantifying the % increase?  No need to share hard numbers, but can we share improvement %?

james-vendia replied:
If it's just error handling, lets cut it.  We usually don't get into defect fixes for the public / non-customer version of release notes.

james-vendia commented on releases/2022-10-28-releasenotes.md:
```diff
@@ -0,0 +1,38 @@
+# Ensuring Transaction Success
+We continue to make improvements to the speed of [consensus](https://www.vendia.com/docs/share/terms-and-definitions#consensus) and success rate of transactions by reducing scenarios in which errors might display. This release focused on resolving errors due to empty versions of data, complex access rules (ACLs), duplicate attribute names exist in different nested paths, and various uni deploymen ...
+
+# Keeping the SDK Current
```
Example code here would be good.  Both the generated code (where the default is applied) and the approach to override (if the previous default of `Node_Ledgered` is still desired).

francine-vendia replied:
added!

james-vendia reviewed (changes requested):
A few places to cut, a few places to expand.

Adding in the nested filter example (you can borrow Matt's screenshots from the issue itself) is the big one...and should probably be what we lead with (that and SDK default syncMode change).

francine-vendia commented:
![Filtering_on_Nested_Attribute](https://user-images.githubusercontent.com/105127677/198392310-636b9718-549b-4914-92ad-b7f6056f499e.png)

francine-vendia commented:
@james-vendia - Incorporated the suggested updates and added items from both release-notes-2022-10-27 and the nested filtering. This PR is updated with the latest version.

Note - The `2022-10-27` release had to be rolled back but *may* still get pushed tomorrow. If it does, these can remain as-is. If not, we'll need to remove `# Bug Squashing: Lambda Subscriptions`, the note on `#ref` fix in `# Ensuring Transaction Success` and `# Returning More in Lists`.

Let me know if all looks good and we'll queue it up.

james-vendia reviewed (approved):
Thanks for the updates.  One additional suggested deletion but otherwise LGTM.

james-vendia commented:
May want to get a better version of the image here - https://github.com/vendia/blog/pull/224#issuecomment-1294033551.  Quality looks lower than we'd like for public content, unless it's a GH rendering issue.

francine-vendia commented:
![Filtering_on_Nested_Attribute3](https://user-images.githubusercontent.com/105127677/198698735-28fdc85a-8c0b-441b-a982-2c3e5aeb1062.png)
New image for nested filtering

francine-vendia commented:
@james-vendia  - latest round of updates are in based on what was rollbacked vs released (and removed all references to fixes errors / bug squashing). Also updated the image for nested filtering - let me know if that one looks better.

james-vendia commented:
🚢
